calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
 title: View flow logs
 ---
 
+import Screenshot from '../../src/___new___/components/Screenshot';
+
 # View flow logs in the Calico Whisker web console
+
+You can observe network traffic in your cluster by viewing flow logs in the Calico Whisker web console.
+
+## About flow logs in Calico Open Source
+
+Tracking network activity in your cluster is essential for understanding how your applications communicate.
+Calico helps you see your network activity by processing your network flows and recording them in a digestible format called a flow log.
+
+A flow log isn't a record of every single network connection that takes place in your cluster.
+Rather, it is an aggregated summary of a type of network flow that occurs in your cluster over a period of time.
+Imagine, for example, that your cluster has 1000 individual network flows that take place within 1 minute.
```

Comment:
```suggestion
Imagine, for example, that your cluster has 1000 individual network connections that take place within 1 minute.
```

Same thing here - I think it's confusing to define a flow log in terms of flows, because we haven't defined what a flow is.